mappings/net/minecraft/item/FoodItemSetting.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS awp net/minecraft/item/FoodItemSetting
+	CLASS awp$a Builder
+		FIELD a food I
+		FIELD b saturation F
+		FIELD c wolfFood Z
+		FIELD d alwaysEdible Z
+		FIELD e eatenFast Z
+		FIELD f statusEffectChances Ljava/util/List;
+		METHOD a wolfFood ()Lawp$a;
+		METHOD a setSaturation (F)Lawp$a;
+			ARG 1 amount
+		METHOD a setFood (I)Lawp$a;
+			ARG 1 amount
+		METHOD a addStatusEffect (Lahq;F)Lawp$a;
+			ARG 2 chance
+		METHOD b alwaysEdible ()Lawp$a;
+		METHOD c eatenFast ()Lawp$a;
+		METHOD d build ()Lawp;
+	FIELD a food I
```

Comment:
this and any other place it says "food" should probably say "hunger", like with the old FoodItem stuff.

Reply:
Would renaming the fields to "hunger" and getters/setters in `FoodItemSetting` to getHunger/setHunger do it? Since some methods that I made contain "food" make sense to be called like that, for example LivingEntity#eatFood and LivingEntity#applyFootEffects. And calling the class `FoodItemSetting` makes sense.

Reply:
Yeah, that's what I mean. Just anywhere where it's talking about hunger restoration.